README.md

```diff
@@ -117,27 +117,16 @@ These tasks are preconfigured
 
 To test changes from packages in this repository in other applications follow these steps:
 
-1. Run `npm install` in the root of the `frontend-components` working copy
-2. Remove `react` and `react-dom` from `node_modules`
-  ```
-  rm -rf node_modules/react; rm -rf node_modules/react-dom
-  ```
-  This is because we want to use hooks and different reacts are not playing nicely with hooks [facebook/react/issues/15315](https://github.com/facebook/react/issues/15315)
-
-3. Link `react` and `react-dom` from your application. Running from folder that contains your application and frontend components. Running `ls` in this folder would yield `<application-folder>  insights-proxy  frontend-components`
-```
-ln -s $PWD/<application-folder>/node_modules/react frontend-components/node_modules/react
-ln -s $PWD/<application-folder>/node_modules/react-dom frontend-components/node_modules/react-dom
-```
-
-4. Change into the directory of the package you are working on, for example `cd packages/components` and run `npm link`*
-5. Change into the directory of the application you'd like to include the package and run `npm link @redhat-cloud-services/frontend-components`*
+1. Install [yalc](https://github.com/whitecolor/yalc) globally. e.g. `npm install -g yalc`.
+2. Run `npm install` in the root of the `frontend-components` working copy.
+3. Change into the directory of the package you are working on, for example `cd packages/components` and run `yalc publish`*
```

Comment:
The `yalc publish` documentation mentions it will "copy" assets that would normally be pushed, does this mean that `publish` needs to be run anytime assets are rebuilt when running `rpm start`?

Reply:
That's correct, it doesn't have a built-in `watch mode`. You can use something like `watch -n 0.5 yalc publish --push --changed` to push assets only when there are changes.

Reply:
Can you include that in the docs as well? I like this solution!

Reply:
Yes, I'll work on it now.

Reply:
Done